### DISCUSSION TITLE
Support Request - profile removal for privacy reasons

### DISCUSSION THREAD

**Discussion by Habiba-Abdelrehim** [yhype/yhype-feedback#18] [General]
Hi YHYPE support, would you remove my profile from YHYPE ? for personal privacy reasons. Would be appreciated :) Thanks

**Comment by antonkomarev** [comment]
We can remove private info only. Public info is publicly available and could be parsed again from other user requests.

**Reply by Habiba-Abdelrehim** [reply]
@antonkomarev ok so the problem is that I removed my profile picture from github but its still showing up when I search my name with yhype. The photo shows up in google images but when I click on the site its removed. Are you able to remove it from the google search ?

**Reply by antonkomarev** [reply]
It is impossible, because it's Google data. I suppose that they scrap web data periodically and will renew it then.

**Reply by Habiba-Abdelrehim** [reply]
I see. So you recommend i should just wait for them to renew the data ?

**Reply by antonkomarev** [reply]
Yes, because we cannot say Google to remove part of the content on the page :)

**Reply by antonkomarev** [reply]
@Habiba-Abdelrehim is problem solved?
